corpus/transform.py

```diff
@@ -293,9 +294,11 @@ def info(self):
 
 
 class MergeStrategy(enum.Enum):
-    SUBCORPORA = 0
-    FLAT = 1
-    CONCATENATE = 2
+    SUBCORPORA = 0  # Merge as subcorpora of a common corpus.
+    FLAT = 1  # Flatten corpus structure by ignoring subcorpora.
+    CONCATENATE = 2  # Concatenate all subcorpora, recordings and speakers.
+    # Like SUBCORPORA, but flatten each top level subcorpora, and merge subcorpora and recordings with the same name.
+    MERGE_SUBCORPORA_AND_RECORDINGS = 3
```

Comment:
I don't think it is "like SUBCORPORA" as in SUBCORPORA we add one layer of subcorpus-ness and here we do merging
```suggestion
    MERGE_SUBCORPORA_AND_RECORDINGS = 3  # Merge subcorpora and recordings with the same name.
```